cgstats/db/cli.py

```diff
@@ -149,10 +153,14 @@ def add(context, machine, demux_dir, unaligned, all_unaligned):
             click.echo(click.style("Please specify an unaligned directory for NovaSeq!", fg='yellow'))
             context.abort()
         if all_unaligned:
-            unaligned_dirs = glob(Path(demux_dir).joinpath('Unaligned*'))
+            unaligned_dirs = glob(demux_dir_path.joinpath('Unaligned*'))
             for unaligned in unaligned_dirs:
                 log.info('Adding {}.'.format(unaligned))
                 novaseqparse.add(manager, demux_dir, unaligned, machine)
+        if machine == '2500':
+            if check_for_rapid_support(unaligned_dir_path=unaligned_dir_path):
+                log.info('Adding {}.'.format(unaligned))
+                parse.add(manager, demux_dir, unaligned)
```

Comment:
I think we can avoid some nesting by going:
```
if machine == 'novaseq' and not unaligned:
    do_something()
if machine == 'novaseq'  and all_aligned:
    do_something_else()
if machine == 'novaseq' and unaligned:
    do_a_thing()
if machine == '2500':
    check_for_stuff()
``` 
This feels a lot more readable to me, what do you think?

Reply:
Since I already merged this branch into master no updates are getting showed here. You can see my changes in https://github.com/Clinical-Genomics/cgstats/blob/fix-add-rapid-flowcells/cgstats/db/cli.py